packages/core/src/babel-plugin-redwood-import-dir.ts

```diff
@@ -47,17 +47,30 @@ export default function ({ types: t }: { types: typeof types }): PluginObj {
 
         const cwd = path.dirname(state.file.opts.filename)
         const dirFiles = glob.sync(p.node.source.value, { cwd })
-        for (const filePath of dirFiles) {
+
+        const sourceFilesOnly = dirFiles.filter(
+          (dirFile) => !dirFile.includes('.test.')
+        )
+        for (const filePath of sourceFilesOnly) {
           const fileName = path.basename(filePath).split('.')[0]
           // + import * as <importName>_<fileName> from <filePath>
+
+          const parsedPath = path.parse(filePath)
+
+          // Do it this way to allow double dots e.g. service/payment/payment.utils.ts
+          const filePathWithoutExtension = path.join(
+            parsedPath.dir,
+            parsedPath.name
+          )
```

Comment:
node imports paths are normalized across operating systems, so I don't think we need to use `path.join()` here, instead `.join('/')` should be enough.

Reply:
👍 Change coming up